packages/plugin-typescript/src/lib/runner/utils.ts

```diff
@@ -0,0 +1,107 @@
+// eslint-disable-next-line unicorn/import-style
+import { dirname } from 'node:path';
+import {
+  type Diagnostic,
+  DiagnosticCategory,
+  flattenDiagnosticMessageText,
+  parseConfigFileTextToJson,
+  parseJsonConfigFileContent,
+  sys,
+} from 'typescript';
+import type { Issue } from '@code-pushup/models';
+import { readTextFile, truncateIssueMessage } from '@code-pushup/utils';
+import { TS_CODE_RANGE_NAMES } from './ts-error-codes.js';
+import type { CodeRangeName } from './types.js';
+
+/**
+ * Transform the TypeScript error code to the audit slug.
+ * @param code - The TypeScript error code.
+ * @returns The audit slug.
+ * @throws Error if the code is not supported.
+ */
+export function tSCodeToAuditSlug(code: number): CodeRangeName {
```

Comment:
Typo in function name:

```suggestion
export function tsCodeToAuditSlug(code: number): CodeRangeName {
```